lib/usecases/run/GetAllRunsUseCase.js

```diff
@@ -78,16 +78,33 @@ class GetAllRunsUseCase {
 
             if (runNumbers) {
                 const runNumberList = runNumbers.split(SEARCH_ITEMS_SEPARATOR)
```

Comment:
This should be renamed to `runNumberCriteria`, and your list `allRuns` should be something like `runNumberSet` (allRuns would mean runs entities)

Reply:
Done